pages/common/nano.md

```diff
@@ -3,10 +3,14 @@
 > Simple, easy to use editor. An enhanced, free Pico clone.
 > More information: <https://nano-editor.org>.
 
-- Start nano in terminal with {filename}:
+- Start nano in terminal with a specified filename:
```

Comment:
```suggestion
- Start nano opening a specific file:
```

Reply:
Thinking about this, it probably isn't necessary to mention nano in each example. So they could probably just be:

- Open a specific file:
- Open a file with the cursor at a specific line:
- etc.

Reply:
@pxgamer true, updated.